.github/workflows/buildsCI.yaml

```diff
@@ -400,4 +400,34 @@ jobs:
        cmake --build . --config ${{env.LRS_BUILD_CONFIG}} -- -j4
        ls
   
-  
+  cppcheck:
+    name: cppcheck-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+          
+      - name: cppcheck_run
+        uses: deep5050/cppcheck-action@main
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN}}
+          std: c++11
+          max_ctu_depth: 4
```

Comment:
Please add a comment what does it means..

Reply:
It supposed to perform a deeper scan but I deleted because the result was no difference than the default